models.go

```diff
@@ -38,6 +38,21 @@ type App struct {
 	UpdatedAt   time.Time
 }
 
+type SumResult struct {
+	Sum uint
+}
+
+type AppPermission struct {
+	ID                     uint `gorm:"primaryKey"`
+	AppId                  uint `gorm:"index" validate:"required"`
+	App                    App  `gorm:"constraint:OnDelete:CASCADE"`
+	NostrKind              int  `gorm:"index" validate:"required"`
+	MaxAmount              int
```

Comment:
If we wanted to include fees, I wonder if this should be an optional setting. Especially if there are routing fee limits already set (e.g. in Alby 1% or on your node) maybe it is not necessary?

Reply:
Added note that fees are currently not included